starter-code/pizza.js

```diff
@@ -1 +1,95 @@
 // Write your Pizza Builder JavaScript in this file.
+
+$(document).ready(function() {
+  $('.green-pepper').toggle();
+  $('.pep').toggle();
+  $('.stem').toggle();
+  $('.cap').toggle();
+  $('.sauce-white').toggle();
+});
+
+var priceString = $('strong').html();
+var totalPrice = Number(priceString.slice(1));
+
+$('.btn').click(function () {
+  $(this).toggleClass('active');
+});
+
+
+$('.btn-pepperonni').click(function () {
+  var display = $('aside ul li').eq(0).css('display');
+  var pep = $('aside ul li').eq(0);
+  if (display === 'none') {
+    totalPrice += 1;
+  } else {
+    totalPrice -= 1;
+  }
+    $('.pep').toggle();
+    toggleLIElements(0);
+    $('strong').html(totalPrice);
+});
+
+function toggleLIElements(value) {
+  var display = $('aside ul li').eq(value).css('display');
+  var pep = $('aside ul li').eq(value);
+
+  if (display === 'none') {
+    pep.css('display', 'block');
+  } else {
+    pep.css('display', 'none');
+  }
+}
+
+$('.btn-mushrooms').click(function () {
+  var display = $('aside ul li').eq(1).css('display');
+  var pep = $('aside ul li').eq(1);
+  if (display === 'none') {
+    totalPrice += 1;
+  } else {
+    totalPrice -= 1;
+  }
+    $('strong').html(totalPrice);
+    $('.stem').toggle();
+    $('.cap').toggle();
+    toggleLIElements(1);
+  });
+
+$('.btn-green-peppers').click(function () {
+  var display = $('aside ul li').eq(2).css('display');
+  var pep = $('aside ul li').eq(2);
+  if (display === 'none') {
+    totalPrice += 1;
+  } else {
+    totalPrice -= 1;
+  }
+    $('strong').html(totalPrice);
+    $('.green-pepper').toggle();
+    toggleLIElements(2);
+  });
+
+$('.btn-sauce').click(function () {
+    $('.sauce-white').toggle();
+    var display = $('aside ul li').eq(3).css('display');
+    var pep = $('aside ul li').eq(3);
+    if (display === 'none') {
+      totalPrice += 3;
+    } else {
+      totalPrice -= 3;
+    }
+      $('strong').html(totalPrice);
+    toggleLIElements(3);
+  });
+
+$('.btn-crust').click(function () {
+    $('.sauce-white').toggle();
+    var display = $('aside ul li').eq(4).css('display');
+    var pep = $('aside ul li').eq(4);
+    if (display === 'none') {
+      totalPrice += 5;
+    } else {
+      totalPrice -= 5;
+    }
+      $('strong').html(totalPrice);
+    $('.crust-gluten-free').toggleClass('crust');
+    toggleLIElements(4);
+  });
```

Comment:
You guys got creative in the way you selected the elements which is always a great thing.

I feel like you guys may appreciate another solution or different perspectives to problems so I'll add this here:

//For the click functions of all of the buttons you could have done something like this.

//Click event handler for btn-pepperonni
  $('.btn-pepperonni').click(function(e){
  //Fills pizza with pepperoni
  $('.pep').toggle(500);
 //Toggles the 'active' class on or off.
 //If class does not exist, it will be created and then switched on and off.
 $(this).toggleClass('active');
 
  //Checks to see if button presses is active.
 //If true, base price of pizza is incremented.
 //If false, price of pizza decreases.
 if ($(this).hasClass('active')){
      pizzaBasePrice += 1;
      $('strong').html("$" + pizzaBasePrice);
     }
     else{
       pizzaBasePrice -= 1;
       $('strong').html('$' + pizzaBasePrice);
     }
     //Displays added ingredient in the price list.
     $('.price > ul > li:nth-child(1)').toggle();
   });

// This you may have saved you some work.